client/state/selectors/is-sharing-buttons-save-successful.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Returns true if the save sharing buttons requests is successful
+ *
+ * @param  {Object}  state  Global state tree
+ * @param  {Number}  siteId Site ID
+ * @return {Boolean}        Whether the request is successful or not
+ */
+export default function isSiteSettingsSaveSuccessful( state, siteId ) {
```

Comment:
This will be used in one place.

In a previous PR (Reduxify Site Settings #9056) @aduth proposed to have this, thus, the component won't have to know the value of the status. I think we should keep it consistent between the two sub-states at least.